### DISCUSSION TITLE
RG350p: left analog stick not enabled for NES and SNES games

### DISCUSSION THREAD

**Discussion by Blawkowski** [eduardofilo/RG350_adam_image#155] [Ideas & Requests]
I can use left analog stick in Intellivision games but not for NES and SNES.

I could enable it in Retroarch options but config won't stick when I exit a game and having it enabled by default would be nice.

**Comment by eduardofilo** [comment]
It is necessary to make an override to keep the change. I will consider adding it to the image. Thanks.

**Comment by eduardofilo** [comment]
My previous comment was not correct. If the adjustment is made within the core options, the changes are not saved with an override, but with a remap file (menu `Quick Menu > Controls > Save Core Remap File`. However, I will include these adjustments in a future image release.

**Reply by Blawkowski** [reply]
Hi eduardofilo,

Thanks for the tip, it worked!

Can't wait to test your next release!
